core/embed/rust/src/ui/geometry.rs

```diff
@@ -241,6 +252,10 @@ impl Rect {
         }
     }
 
+    pub fn from_size(size: Offset) -> Self {
```

Comment:
could this be const? also applies to bunch of functions below

Reply:
Yes, of course, it's possible, fixed 2536ed1b7195001706689a843df0cfd43b08340d